semcore/mini-chart/src/component/score/donut.shadow.css

```diff
@@ -0,0 +1,17 @@
+@import '../skeleton/skeleton.shadow.css';
+
+SDonutContainer[semi] {
+  g {
+    transform: rotate(180deg) translate(-24px, -24px);
```

Comment:
What means the magic numbers 24?

Reply:
this is a bias because of radius and stroke width